### PR TITLE
adds option to disable developers news on start page (fixes #2 )

### LINKED ISSUE #2: [FEATURE] Implement option to disable developer news
**Is your feature request related to a problem? Please describe.**

Add option to disable developer news in StartPage+

**Describe the solution you'd like**

Add option to disable developer news, people who want it back can easily install a [VS2022 compatible version from the market place](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.DeveloperNews2022), for me it only wastes space that I would like to get filled with larger paths from the recent menu.

**Describe alternatives you've considered**

People who want it back can easily install a [VS2022 compatible version from the market place](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.DeveloperNews2022) 

image

### PR DESCRIPTION
**Pull request by MagicAndre1981**
allowing users to turn off developers news ( fixes #2 )

### CODE REVIEW

**Comment by yannduran** [comment]
There are a couple of changes that need to be made, but you haven't enabled me to make any changes to the pull request.

**Comment by MagicAndre1981** [comment]
> but you haven't enabled me to make any changes to the pull request.

I did: 

image

image

**Comment by MagicAndre1981** [comment]
if you still can't commit, close it and do it on your own. I only tried to show that is very easy

**Comment by yannduran** [comment]
OK, I'll do that. I'll try to make the changes again. You did solve one of the problems that I have a vague memory of. 

If not I'll do as you suggest and do it myself. I just didn't want to not accept a generously offered pull request.

Thanks André!

**Comment by yannduran** [comment]
I can provide you with the experience that you've been able to achieve yourself so far, of course. But the main problem that remains is **replacing** the `UniformDock` control with something that will still give users who do want all 3 panes the same experience as now. Mainly for if they change the `MaximumWidth` property, or move from one monitor to another that uses a different ratio etc.

I have a few ideas I want to try, but that'll probably be sprint v0.23, or sooner if I'm able to get through things quicker.

File: src/vsix/UI/ViewModels/NewsItems/NewsItemsViewModel.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public NewsItemsViewModel(INewsItemDataService dataService, INewsItemCommandServ
             _visualStudioService = visualStudioService;
 
             Heading = HEADING;
-            IsVisible = true;
+            IsVisible = NewsItemsOptions.Instance.DisplayNewsItems;
```
Comment (yannduran) [inline src/vsix/UI/ViewModels/NewsItems/NewsItemsViewModel.cs]: This isn't necessary

File: src/vsix/UI/ViewModels/StartViewModel.cs [hunk 2]
```diff
@@ -15,12 +15,23 @@ public StartViewModel(RecentItemsViewModel recentItemsViewModel, StartItemsViewM
...
-            };
+                Columns = new ObservableCollection<ColumnViewModel>
+                {
+                    recentItemsViewModel,
+                    startItemsViewModel,
+                    newsItemsViewModel
+                };
+            }
+            else
+            {
```
Comment (yannduran) [inline src/vsix/UI/ViewModels/StartViewModel.cs:28]: I hadn't even thought to approach it this way. I think that's why I didn't finish it years ago. I just made the column invisible, and of course it was invisible, but was still there on the page.

**Comment by yannduran** [comment]
Being a one-man shop developer, I've never needed to use pull requests. It's always just been me How do I add code. Do I have to clone the public repo and do it in the pull request there, then push it back up for you to do whatever you need to do and then I approve it? 

Is that how it works? I don't mind learning about pull requests though.

**Comment by yannduran** [comment]
>Add more commits by pushing to the [issue2disabledevelopernews](https://github.com/MagicAndre1981/start-page-plus-64-bit/tree/issue2disabledevelopernews) branch on [MagicAndre1981/start-page-plus-64-bit](https://github.com/MagicAndre1981/start-page-plus-64-bit).

Ahh, there was a line there telling me what to do lol

**Comment by yannduran** [comment]
Added in v0.22

**Pull request closed without merging**